### ISSUE TITLE
Design: The People Page

### ISSUE THREAD

**Issue by BrianL3** [CouncilDataProject/cdp-frontend#164]
### The People Page

The People Page needs two things:

1. a design
2. someone to implement that design

### Use Case

Part of this story is to have the design team come up with the different use cases of the People Page. Remember that we have a number of different user archetypes: the journalist, the academic researcher, and the layperson (average citizen). 
Is a citizen coming to this page because they do not know who their current council members are?
Does the user want to look for a specific person?
Which people are most important to show?
Are there any other considerations for searching, browsing, or perhaps "custom" looks ("show me who was on the council in XYZ year")?

The primary ways we have of looking at people are Role, time and geography.

- For more on what roles are available, talk to @JacksonMaxfield .
- Some Roles have Seats (e.g. if you are the Councilperson of the Beacon Hill neighborhood, your Seat would be there). So there is an aspect of geography available for some roles. 
- Time could be an interesting aspect, as CDP does have use as a research tool.

### Solution

The PersonCard (see the storybook library to see this component) was designed to be used on this page. 

### Alternatives

The current PeoplePage as shown is an alternative, I suppose.  It is just a list of Person links by name. I think we can all agree that it needs a re-vamp.

### Current Look Reference
![Screen Shot 2021-09-24 at 4 57 32 PM](https://user-images.githubusercontent.com/9027758/148597802-ab1705d1-c0a7-4921-9bad-355c8f9dc25e.png)

**Comment by tohuynh** [comment]
@BrianL3 Is this for the `/people` page, not the `/people/:id` page right?

**Comment by BrianL3** [comment]
Yes, this is for the /people page.  Sorry, I uploaded the wrong thing.
